### PR TITLE
docs(PPDSC-2289): Text block documentation

### PR DESCRIPTION
**Pull request by Srinivasan-Ramasamy**
PPDSC-2289

**What**

1. Text block documentation UI - http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2289-text-block-documentation/components/text-block/

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [X] Updated relevant documentation

**I have tested manually:**
- [X] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [X] The screen reader reads and flows through the elements as expected.
- [X] There are no new errors in the browser console coming from this PR.
- [X] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [X] The Playground feature is working as expected

**Before:**

**After:**

image

**Who should review this PR:**

**How to test:**

URL: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2289-text-block-documentation/components/text-block/

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2289-text-block-documentation
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2289-text-block-documentation/storybook

**Review by mstuartf — APPROVED** [review]

File: site/pages/components/text-block.tsx [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+                'If provided, this overrides the style preset applied to the text block.',
+            },
+            {
+              name: 'typographyPreset',
+              type: 'MQ<string>',
+              description:
+                'If provided, this overrides the typography preset applied to the text block.',
+            },
+          ],
+        },
```
Comment (Xin00163) [inline site/pages/components/text-block.tsx:303]: TextBlock now support LogicalProps. Could we please add these to the docs?

**Comment by Srinivasan-Ramasamy** [comment]
Thanks @mstuartf and @Xin00163 for taking the time to review :+1:

File: site/pages/components/text-block.tsx (hunk 1 above)
Comment (Srinivasan-Ramasamy) [reply site/pages/components/text-block.tsx:306]: Thanks @Xin00163m, updated the changes. Please review it once.

**Pull request merged**